spec/features/traject/ead2_indexing_spec.rb

```diff
@@ -87,6 +87,11 @@
       expect(result['places_ssim']).to eq ['Yosemite National Park (Calif.)']
     end
 
+    it 'has_online_content' do
+      pp result
```

Comment:
Perhaps leftover debuggin?

Reply:
whoops, yes, that is what that is. When there is this kind of small change suggested, and then the PR is merged, what is the appropriate action for me to take?

Reply:
Maybe a submit another PR to remove the debug?

I didn't want y'all to have to wait another day to get this shipped.